app.js

```diff
@@ -0,0 +1,86 @@
+// import functions and grab DOM elements
+import { getRandThrow, testIfWin } from '/utils.js';
+
+const playButton = document.getElementById('play-button');
+const resetButton = document.getElementById('reset-button');
+const winsDisplay = document.getElementById('times-won');
+const totalsDisplay = document.getElementById('times-played');
+const tiesDisplay = document.getElementById('times-draw');
+const lossDisplay = document.getElementById('times-lost');
+const compDisplay = document.getElementById('comp-threw');
+const playerDisplay = document.getElementById('player-threw');
+
+// initialize state
+//default string values to display when user clicks reset button
+const defaultStrings = [
+    'Wins go here, if you win at all muahaha', 
+    'Plays go here if you dare to go again', 
+    'If you TIED its because I didnt try', 
+    'Times lost goes here, RIPieces'];
+
+// initialize counts for totals
+let totalPlays = 0;
+let totalWins = 0;
+let totalTies = 0;
+let totalLosses = 0;
+
+// set event listeners to update state and DOM
+function convertCompNumToValue(getRandThrow){
```

Comment:
You need to call the function getRandThrow() <- parentheses missing. Also you might need to set it to a const first.